### PR TITLE
Some minor changes/fixes to ux backend

### PR DESCRIPTION
**Pull request by nb-ohad**
- Move content of services/types package to the services package
- Move onboading_tokens content into its own sub-package
- Separate msg handling into Distinct functions based on HTTP method
- fix onboarding_tokens invalid content type

Signed-off-by: nb-ohad <mitrani.ohad@gmail.com>

### CODE REVIEW

File: services/ux-backend/handlers/onboarding_tokens/handler.go [hunk 1]
```diff
@@ -15,64 +15,63 @@ import (
...
-		_, err = w.Write([]byte(fmt.Sprintf("Unsupported method : %s", r.Method)))
-		if err != nil {
-			klog.Errorf("failed write data to response writer: %v", err)
-		}
-		return
+func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
+	klog.Info("Only POST method should be used to send data to this endpoint /onboarding-tokens")
+	w.WriteHeader(http.StatusMethodNotAllowed)
+	w.Header().Set("Content-Type", ContentTypeTextPlain)
+
```
Comment (leelavg) [inline services/ux-backend/handlers/onboarding_tokens/handler.go]: minor: MDN [docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) says `server must generate an Allow header field in a 405 status code response`

File: services/ux-backend/handlers/onboarding_tokens/handler.go [hunk 2]
```diff
@@ -15,64 +15,63 @@ import (
...
-const onboardingPrivateKeyFilePath = "/etc/private-key/key"
-
-func OnboardingTokensHandler(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
+const (
+	onboardingPrivateKeyFilePath = "/etc/private-key/key"
+	ContentTypeTextPlain         = "text/plain"
+)
 
-	var err error
+func HandleMessage(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
```
Comment (leelavg) [inline services/ux-backend/handlers/onboarding_tokens/handler.go]: HandleRequest might sound and suit better for what we are doing 🤔

File: services/ux-backend/handlers/onboardingtokens/handler.go [hunk 3]
```diff
@@ -15,64 +15,63 @@ import (
...
-const onboardingPrivateKeyFilePath = "/etc/private-key/key"
-
-func OnboardingTokensHandler(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
+const (
+	onboardingPrivateKeyFilePath = "/etc/private-key/key"
+	ContentTypeTextPlain         = "text/plain"
+)
 
-	var err error
+func HandleMessage(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
```
Comment (leelavg) [inline services/ux-backend/handlers/onboardingtokens/handler.go:27]: `HandleRequest` might sound and suit better for what we are doing 🤔

File: services/ux-backend/handlers/onboardingtokens/handler.go [hunk 4]
```diff
@@ -15,64 +15,63 @@ import (
...
-		_, err = w.Write([]byte(fmt.Sprintf("Unsupported method : %s", r.Method)))
-		if err != nil {
-			klog.Errorf("failed write data to response writer: %v", err)
-		}
-		return
+func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
+	klog.Info("Only POST method should be used to send data to this endpoint /onboarding-tokens")
+	w.WriteHeader(http.StatusMethodNotAllowed)
+	w.Header().Set("Content-Type", ContentTypeTextPlain)
+
```
Comment (leelavg) [inline services/ux-backend/handlers/onboardingtokens/handler.go:60]: MDN [docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) says `server must generate an Allow header field in a 405 status code response`

**Comment by leelavg** [comment]
/hold

refer #2352

File: services/ux-backend/handlers/onboardingtokens/handler.go (hunk 3 above)
Comment (nb-ohad) [reply services/ux-backend/handlers/onboardingtokens/handler.go:27]: For me, Request is the vehicle, Message is the content we want to handle.

File: services/ux-backend/handlers/onboardingtokens/handler.go (hunk 4 above)
Comment (nb-ohad) [reply services/ux-backend/handlers/onboardingtokens/handler.go:61]: done

**Comment by rchikatw** [comment]
LGTM, Please fix the commit lint check

**Pull request merged**